src/hooks/prerun/install-prerun.js

```diff
@@ -1,10 +1,19 @@
 /* eslint-disable no-console */
 const chalk = require('chalk');
 
-const UPDATE_ALERT = `Warning: Use '${chalk.bold('npm update twilio-cli')}' for npm based installations. `;
+const urlUtil = require('../../services/hyperlink-utility');
+
+const TWILIO_CLI_DOCS_LINK = 'https://www.twilio.com/docs/twilio-cli/quickstart';
+let UPDATE_ALERT = `Warning: Use '${chalk.bold('npm update twilio-cli')}' for npm based installations. `;
```

Comment:
Do we need to update this alert also now that we have other methods? Or completely remove it since we are pointing to the docs?

Reply:
Yeah makes sense with added new methods, updated the text to be generic.